src/demos/select-field/select-field-demo-custom-picker.component.html

```diff
@@ -0,0 +1,50 @@
+<sky-modal>
+  <sky-modal-header>
+    {{ title }}
+  </sky-modal-header>
+  <sky-modal-content>
+    <ng-container
+      *ngIf="context.pickerType === 'color'; else fruitPicker"
```

Comment:
Code style > Structural directives should be on the same line (I see a few of those in this template).